package.json

```
@@ -8,28 +8,30 @@
     "@testing-library/user-event": "^12.1.10",
     "@types/jest": "^26.0.15",
     "@types/node": "^12.0.0",
-    "@types/react": "^16.9.53",
-    "@types/react-dom": "^16.9.8",
+    "@types/react": "^18.0.14",
+    "@types/react-dom": "^18.0.5",
     "apisauce": "^1.1.2",
     "history": "^4.10.1",
-    "i18next": "^13.1.5",
+    "i18next": "^19.9.2",
     "node-sass": "^4.14.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-hook-form": "^6.14.2",
-    "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-hook-form": "^7.33.0",
+    "react-i18next": "^11.17.3",
+    "react-router": "^6.3.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "^4.0.3",
     "react-spinkit": "^3.0.0",
-    "typescript": "^4.1.3",
-    "web-vitals": "^0.2.4",
+    "typescript": "^4.3.2",
+    "web-vitals": "^1.0.1",
     "wolox-equalizer": "0.0.3"
   },
   "scripts": {
     "start": "rescripts start",
     "start-env": "node ./scripts/start.js",
     "build": "node ./scripts/build.js",
-    "test": "rescripts test --env=jsdom",
+    "test": "rescripts test --env=jsdom --passWithNoTests --watchAll=false",
+    "ts-check": "tsc",
     "eject": "./node_modules/react-scripts/bin/react-scripts.js eject",
     "lint": "./node_modules/eslint/bin/eslint.js src --ext .js --ext .ts --ext .tsx",
     "lint-fix": "npm run lint -- --fix && ./node_modules/stylelint/bin/stylelint.js '**/*.scss' --fix",
```

This have problemas on Windows. Show a problem error as `"." no se reconoce como un comando interno o externo`. We fix this problem like this: 
```suggestion
    "eject": "./node_modules/react-scripts/bin/react-scripts.js eject",
    "lint": "node ./node_modules/eslint/bin/eslint.js src --ext .js --ext .ts --ext .tsx",
    "lint-fix": "npm run lint -- --fix && node ./node_modules/stylelint/bin/stylelint.js '**/*.scss' --fix",
```